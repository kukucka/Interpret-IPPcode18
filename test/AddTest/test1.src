.IPPcode18
DEFVAR GF@counter
MOVE GF@counter  string@
#Inicializace  proměnné na prázdný řetězec
#Jednoduchá iterace , dokud  nebude  splněna zadaná podmínka
LABEL while
JUMPIFEQ end  GF@counter  string@
WRITE string@řetězec\032s\032lomí##tkem\032\092\032a\010novým\035řádkem
WRITE GF@counter
WRITE string@\010
CONCAT GF@counter  GF@counter  string@a
JUMP while
LABEL end
MOVE GF@Souther bool@false