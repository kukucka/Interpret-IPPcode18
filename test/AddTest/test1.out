Error varName, var doesnt exist
